eth/core/precompile/method_test.go

```diff
@@ -52,17 +51,6 @@ var _ = Describe("Method", func() {
 			err := methodMissingFunc.ValidateBasic()
 			Expect(err).To(HaveOccurred())
 		})
-
-		It("should error on given abi method", func() {
```

Comment:
@shampoobera I think removing this test is bad, if there are unused abi methods, they it should error.

Reply:
oh I mean the test is bad now, but we shouldnt remove the test you feel (just fix it to actually use a bad one)

i'd also add a test for if there are unused abi methods in the solidity file (a.k.a, not matched to something in the go precompile` it should also error.

Reply:
we already do that in `factories_test.go`

https://github.com/berachain/polaris/pull/869/files#diff-623c4a231b9e56ca6de325b03c827a6c7218c9ee347a7a518acba6861ab6df7cR79